src/model.ts

```diff
@@ -637,6 +639,7 @@ export class GitExtension implements IGitExtension {
           'POST',
           {
             clone_url: url,
+            not_versioning: not_versioning,
```

Comment:
This is a shortcut, when the key is identical to the variable name, you can just provide the variable:

```suggestion
            versioning,
```